### PR TITLE
Invenio-Query-Parser v0.5.0

### PR DESCRIPTION
**Pull request by egabancho**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.5.0 (released 2016-02-29):
+------------------------------------
+
+- Allows match unit to iterate over Sequences, i.e. tuple.
+- Removes distinctions between double and single quotes.
+- Adds support for allowed keywords.
+- Removes nestable keywords (to be re added in the future).
```
Comment (switowski) [inline CHANGES.rst]: typo: `readded`

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
 About
 -----
 
 Search query parser supporting Invenio and SPIRES search syntax.
 
+
+Incompatible changes
+--------------------
+
+- Removes nestable keywords (to be re added in the future).
```
Comment (switowski) [inline RELEASE-NOTES.rst]: typo `readded`

**Comment by switowski** [comment]
LGTM, should we :ship ?

**Comment by jirikuncar** [comment]
Please note our new commit message for **all** inveniosoftware releases: `release: v0.5.0`.

**Pull request merged**